torch/_inductor/runtime/hints.py

```diff
@@ -68,7 +68,7 @@ def instance_descriptor(
     instance_descriptor = collections.namedtuple(  # type: ignore[no-redef]
         "instance_descriptor",
         ["divisible_by_16", "equal_to_1", "ids_of_folded_args", "divisible_by_8"],
-        defaults=[tuple(), tuple(), tuple(), tuple()],
+        defaults=[(), (), (), ()],
```

Comment:
Not sure which one is faster, more readable to be frank
```suggestion
        defaults=[()] * 4,
```